test/es-module/test-importmap.js

```diff
@@ -0,0 +1,121 @@
+// Flags: --expose-internals
+'use strict';
```

Comment:
As import maps are far more likely to be used with ESM, this test should probably be written in ESM so that it’s run by the ESM loader.